kotest-property/src/commonMain/kotlin/io/kotest/property/config.kt

```diff
@@ -41,7 +42,6 @@ fun calculateMinimumIterations(vararg gens: Gen<*>): Int {
 }
 
 fun EdgeConfig.Companion.default(): EdgeConfig = EdgeConfig(
-   determinism = PropertyTesting.edgecasesBindDeterminism,
```

Comment:
Noted, this was what sam meant by removing determinism factor. with this removed we lose the ability to have mixtures in edgecases and samples on bind.

Reply:
It feels odd to have two settings - one for edgecases vs samples in general and one for edgecases vs samples in bind.
What about making this a parameter only to bind ?

Reply:
Also I mentioned in the comments about whether this is really useful, like if you have edgecases available, why wouldn't you always use them in the edgecase bind ?

Reply:
Potentially we should be able to reuse the same option as yhey are pretty much related. However, let's put this up first and introduce it when it's required.

Reply:
Ok, so stick with what we have and tweak later.